lib/support/cli.js

```diff
@@ -231,6 +231,12 @@ module.exports.parseCommandLine = function parseCommandLine() {
       type: 'boolean',
       group: 'firefox'
     })
+    .option('firefox.windowRecorder', {
+      describe: 'Use the Firefox internal window recorder',
```

Comment:
Please include something about "video capture" here.  Should this be in the `video.*` section?